app/parse.py

```diff
@@ -14,5 +20,40 @@ class Course:
     course_type: CourseType
 
 
+def parse_single_course(
+        course_soup: BeautifulSoup,
+        course_type: Any
+) -> [Course]:
+
+    list_courses = []
+    list_description = []
+    for i in course_soup.select("span.typography_landingH3__vTjok"):
+        list_courses.append(i.text)
+    for descr in course_soup.select("div p")[3:]:
+        list_description.append(descr.text)
```

Comment:
you can do all of this by using list comprehension

Reply:
changed the code